group-actors/src/main/java/org/sunbird/service/GroupService.java

```diff
@@ -14,9 +14,10 @@
 
   Map<String, Object> readGroup(String groupId) throws BaseException;
```

Comment:
context need to be passed

Reply:
We use MDC to log traceId in groups, so there is no requirement to pass request context in each method to log traceId